pkg/kubectl/plugins/plugins.go

```diff
@@ -105,6 +127,16 @@ func (f Flag) Validate() error {
 	if strings.Contains(f.Name, " ") {
 		return ErrInvalidFlagName
 	}
+	if f.Type != "" {
```

Comment:
We should make this backwards compatible. If the flag type is missing or empty string, we should take it as string flag by default.